README.md

```diff
@@ -89,9 +94,10 @@ In the case above, a `.svg` file will be generated in your working directory.
 > xmindmark --outputDir ./out <your-xmindmark-file>
 ```
 
-You can use `-o` or `--outputDir` flag to specify the output directory where the generated file will be placed. Both relative and absolute path are supported.
+You can use `-o` or `--outputDir` flag to specify the output directory where the generated file will be placed. Both relative and absolute paths are supported.
```

Comment:
It shall be described here is "Either relative or absolute paths are supported."